src/test/java/redis/clients/jedis/tests/ProtocolTest.java

```diff
@@ -119,4 +120,17 @@ public void nullMultiBulkReply() {
     List<String> response = (List<String>) Protocol.read(new RedisInputStream(is));
     assertNull(response);
   }
-}
\ No newline at end of file
+
+  @Test
```

Comment:
Hmmm.. you're right, even though it's trivial I prefer to leave it this way. Thx for clarifying.